### DISCUSSION TITLE
How to remove the grey bounding box caused in transforms

### DISCUSSION THREAD

**Discussion by sherjy** [Project-MONAI/MONAI#2817] [Q&A] [answered]
Hiii, this is a example picture after post-processing. It seems the target is surrounded by a grey bounding box. I want to remove the grey bounding box by black background. I'm troubled by this for a while since I tried with several monai transformations to remove this box, but the grey part was still left. Could you give some guidance for me. Thanks in advance!!!
Here is a demo picture which has a target CT with a surrounding box part:
![image](https://user-images.githubusercontent.com/23369551/130356969-95f016a5-f82d-49d1-a7d3-d88c6368937d.png)

**Comment by Nic-Ma** [comment, marked as answer]
Hi @sherjy ,

Thanks for your interest here.
I think maybe you can try the `CropForeground` transform of MONAI?

Thanks.

**Reply by sherjy** [reply]
Thanks for your advice!! I found there's some wrong with my preset select_fn value. After modifying it, the problem solves.

**Reply by Nic-Ma** [reply]
Glad to see your good progress!

Thanks.
